cli/tools/registry/mod.rs

```diff
@@ -837,7 +837,9 @@ async fn build_and_check_graph_for_publish(
       colors::yellow("Warning"),
     );
     Ok(Arc::new(graph))
-  } else if std::env::var("DENO_INTERNAL_FAST_CHECK_OVERWRITE").is_ok() {
+  } else if std::env::var("DENO_INTERNAL_FAST_CHECK_OVERWRITE").as_deref()
+    == Ok("1")
```

Comment:
Changed this because it makes it easier to turn on and off during debugging in powershell (because it crazily doesn't have a way to set an env var for a single command)